tests/test_net_connect.js

```diff
@@ -1,109 +1,85 @@
 'use strict'
 
-const assert = require('assert')
 const http = require('http')
-const mikealRequest = require('request')
 const { test } = require('tap')
+const assertRejects = require('assert-rejects')
 const nock = require('..')
 const got = require('./got_client')
 
 require('./cleanup_after_each')()
 
-test('disable net connect is default', function(t) {
+test('disable net connect is default', async t => {
   nock.disableNetConnect()
 
   nock('http://example.test')
     .get('/')
     .reply(200)
 
-  mikealRequest('https://google.com/', function(err, res) {
-    assert(err)
-    assert.strictEqual(
-      err.message,
-      'Nock: Disallowed net connect for "google.com:443/"'
-    )
-    t.end()
-  })
+  await assertRejects(
+    got('https://www.bigcorp.test/'),
```

Comment:
As an alternative, to maybe keep things more consistent, the tests already make use of different subdomains when wanting to test different hosts. 
https://github.com/nock/nock/blob/11dba990de94629189dca1030669f2d6d8d409f9/tests/test_intercept.js#L925-L936

Reply:
Updated! A couple of these can just use `example.test`.